src/components/profile/loansAndReservationsPage/Page.js

```diff
@@ -66,7 +67,17 @@ const LoansAndReservations = ({}) => {
       </Text>
       <section className={styles.section}>
         <div className={styles.titleRow}>
-          <Title type="title5" tag="h3">
+          <Title
+            type="title5"
+            tag="h3"
+            id={encodeString(
+              Translate({
+                context: "profile",
+                label: "debt",
+                requestedLang: "da",
+              })
+            )}
+          >
```

Comment:
Er disse delt op, fordi du regner med at de bliver ret forskellige? For ellers ville det nok være fint at lave en lille hjælpe komponent